packages/components/Button/src/Button.tsx

```diff
@@ -34,19 +34,56 @@ export const buttonLookup = (layer: string, state: IPressableState, userProps: B
   );
 };
 
+export const extractOuterStylePropsAndroid = memoize((style) => {
+  const marginKeys = [
+    'margin',
+    'marginTop',
+    'marginBottom',
+    'marginLeft',
+    'marginRight',
+    'marginStart',
+    'marginEnd',
+    'marginVertical',
+    'marginHorizontal',
+    'start',
+    'end',
+    'left',
+    'right',
+    'top',
+    'bottom',
+  ];
+  const outerStyleProps = {},
+    resetProps = {};
```

Comment:
I think for better readability it's good to have `const resetProps = {};` on a new line

Reply:
Yeah we generally don't instantiate multiple variables in one statement.